adapters/e_volution/usersync.go

```diff
@@ -0,0 +1,12 @@
+package evolution
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewEvolutionSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("evolution", temp, adapters.SyncTypeRedirect)
```

Comment:
Please use the family name `"e_volution"`. It's preferred for the family name to match the bidder name. I know we had some issues earlier with renaming the package name to `evolution` from `e_volution`. Was this related?